code/game/machinery/vending.dm

```diff
@@ -1489,8 +1489,13 @@
 /obj/machinery/vending/theomat/proc/check_NT(mob/user)
 	var/bingo = FALSE
 	if(ishuman(user))
+	
 		var/mob/living/carbon/human/H = user
-		if(is_neotheology_disciple(H))
+
+		if(scan_id == !1) //I have no idea why == 0 didn't work. It just wouldn't. But this did. I don't know why. Thanks Jeff Jefferson.
```

Comment:
```suggestion
		if(!scan_id) //I have no idea why == 0 didn't work. It just wouldn't. But this did. I don't know why. Thanks Jeff Jefferson.
```
Maybe this should work?

Reply:
I tried that initially and it didn't work, as id_scan is defined as 1 initially.

Maybe I tried it while it was lower in the if_else sequence when nothing was working.

I can try again tomorrow morning when I have time.

Reply:
It did work. Thank you. I likely had it further down in the if_else statements which gave me trouble. I'm not coding with anything other than what comes to me in dreams.